tsdb/head_test.go

```diff
@@ -6363,6 +6363,154 @@ func TestHeadAppender_AppendCTZeroSample(t *testing.T) {
 	}
 }
 
+func TestHeadAppender_AppendHistogramCTZeroSample(t *testing.T) {
```

Comment:
nit: In general we've tried to keep tests more generic, see for example `TestWALSampleAndExemplarOrder` or even `TestHeadMinOOOTimeUpdate`. So we keep the testcases in sync.

Reply:
Not sure what is being requested here 🤔. Could you elaborate?

Reply:
It's easier to keep `TestHeadAppender_AppendHistogramCTZeroSample` and `TestHeadAppender_AppendCTZeroSample` in sync interms of testcases if they are merged together and the types (float, integer histogram, float histogram) are just sub-testcases like in the tests a I mentioned. The idea is to avoid someone finding a bug in floats, only updating the float test, but missing that there are histograms...

Reply:
Got it, and good idea! I've just pushed a commit with this change